specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/applicationGateway.json

```diff
@@ -2046,6 +2136,42 @@
         "ruleGroupName"
       ],
       "description": "Allows to disable rules within a rule group or an entire rule group."
+    },
+	"ApplicationGatewayAvailableServerVariablesResult": {
```

Comment:
please add "type": "object" to all the model definitions.

Reply:
Added the "type".